src/Auth0RedirectRuleUtilities.js

```diff
@@ -130,10 +135,11 @@ class Auth0RedirectRuleUtilities {
       throw new Error("Cannot redirect");
     }
 
-    const token = sessionToken || this.createSessionToken();
-    this.context.redirect = {
-      url: `${url}?session_token=${token}`,
-    };
+    if (sessionToken !== false) {
```

Comment:
This is fine, but I kind of wish this were checking an option/configuration setting instead of depending on javascript's false vs. undefined semantics.